src/scss/theme/_button.scss

```diff
@@ -33,7 +33,6 @@
 }
 
 // special overrides
-
 .system-main-block {
```

Comment:
I think we should move this scss into your new _user-login.scss

Also, i'm wondering if 'system-main-block' is only used in the user-login ?